src/main/scala/org/hammerlab/guacamole/DistributedUtil.scala

```diff
@@ -336,8 +350,11 @@ object DistributedUtil extends Logging {
     readsRDDs: PerSample[RDD[MappedRead]],
     lociPartitions: LociMap[Long],
     skipEmpty: Boolean,
-    function: PerSample[Pileup] => Iterator[T],
+    function: (PerSample[Pileup], ContigSequence) => Iterator[T],
     reference: ReferenceGenome): RDD[T] = {
+
+    val referenceBC = readsRDDs.head.context.broadcast(reference)
```

Comment:
Since it's a recurring pattern, I won't mention it after this time, but I think we probably don't want to re-broadcast the reference every time we call a pileup function (unless Spark has an optimization for this that I don't know about)